docs/fundamentals/setup-tooling.md

```diff
@@ -169,7 +169,7 @@ These templates matched your input: 'aspire'
 Template Name                     Short Name              Language  Tags
 --------------------------------  ----------------------  --------  -------------------------------------------------------
 .NET Aspire App Host              aspire-apphost          [C#]      Common/.NET Aspire/Cloud
-.NET Aspire Application           aspire                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
+.NET Aspire project           aspire                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
```

Comment:
This is the template name. Needs to remain as is...

```suggestion
.NET Aspire Application           aspire                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
```

Reply:
nice one... we should rename this...... I will send out an email

creates an app host and service defaults... doesn't seem super necessary ..